tests/set_maintenance/test_set_maintenance.py

```diff
@@ -41,6 +41,16 @@ def pulp_client(self):
         return self.pulp_client_controller.client
 
 
```

Comment:
Could we please have at least one of the tests in here have a look at [repo_lock_history](https://release-engineering.github.io/pubtools-pulplib/api/testing.html#pubtools.pulplib.FakeController.repo_lock_history) to confirm that a lock and unlock happened?